### PR TITLE
Adding tags uploader, tags within creation of asset

### PR DESCRIPTION
**Pull request by verenagirgis**

### CODE REVIEW

**Comment by marcomagdy** [comment]
@verenagirgis can you revert the changes to the backend?

File: Properties/launchSettings.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "psaltos": {
       "commandName": "Project",
       "launchBrowser": true,
-      "applicationUrl": "http://localhost:5279",
+      "applicationUrl": "https://localhost:7226;http://localhost:5279",
```
Comment (AthanIsaac) [inline Properties/launchSettings.json:14]: why do we need 2?

File: ClientApp/src/AddData.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    { menuItem: 'Upload Tag', render: () => <Tab.Pane attached={false}>{<UploadTags/>}</Tab.Pane> }
+];
+
+const AddData = () =>  {
+    return (
+        <Tab menu={{ secondary: true, pointing: true }} panes={panes}/>
+    );
+}
+
+export default AddData;
```
Comment (AthanIsaac) [inline ClientApp/src/AddData.js:17]: NIT: NL

File: ClientApp/src/BackendCalls.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        }
+        setAddedToDb(true);
+      })
+      .catch((error) => {
+          console.log(error);
+          setError(true);
+          setErrorMessage("We are able to upload to our database at this time.");
+      });
+
+}
```
Comment (AthanIsaac) [inline ClientApp/src/BackendCalls.js:147]: NIT: NL

File: ClientApp/src/components/PsaltosLoader.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  <Segment>
+    <Dimmer active>
+      <Loader />
+    </Dimmer>
+
+    <Image src='https://react.semantic-ui.com/images/wireframe/short-paragraph.png' />
+  </Segment>
+)
+
+export default PsaltosLoader;
```
Comment (AthanIsaac) [inline ClientApp/src/components/PsaltosLoader.js:14]: NIT: NL

File: ClientApp/src/components/UploadTags.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        <Message hidden={!isError} negative>
+          <Icon name='exclamation circle' loading />
+          <Message.Header>An Error Has Occured</Message.Header>
+          <Message.Content>{errorMessage}</Message.Content>
+        </Message>
+      </div>
+    );
+}
+
+export default UploadTags;
```
Comment (AthanIsaac) [inline ClientApp/src/components/UploadTags.js:53]: NIT: newline

**Comment by AthanIsaac** [comment]
Can you cleanup the code related to the proxy?

**Pull request closed without merging**